C/CutBranching/build_tarballs.jl

```diff
@@ -27,6 +32,7 @@ cmake --install build
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 include("../../L/libjulia/common.jl") 
+#platforms = vcat(libjulia_platforms.(julia_versions)...)
 platforms = supported_platforms()
```

Comment:
Why?  You do _**need**_ to have different builds for different versions of Julia, because Julia internally breaks the ABI in every minor version.

Reply:
Or rather, the question is: why did you had the line commented out?
```julia
#platforms = vcat(libjulia_platforms.(julia_versions)...)
```
That should be uncommented, and you should delete
```julia
platforms = supported_platforms()
```
This PR as-is is not doing anything.

Reply:
Thank you for the feedback. With that line uncommented, the builds did not pass. Now I see that I believe I had to add another line to it. I will update the PR accordingly and see if things now pass